data/json/mapgen/campsite.json

```diff
@@ -70,6 +70,67 @@
       "place_monsters": [ { "monster": "CAMPERS", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 75, "repeat": [ 1, 3 ] } ]
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "campsite" ],
+    "weight": 500,
+    "object": {
+      "faction_owner": [ { "id": "hunters", "x": [ 0, 23 ], "y": [ 0, 23 ] } ],
+      "rows": [
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        ".........###............",
+        "......t..#'#...t........",
+        ".........#+#............",
+        "....###.................",
+        "....#'+....-............",
+        "....###..-ddd-..........",
+        "..........dbd...........",
+        "....###..-ddd-..........",
+        "....#'+....-............",
+        "....###.................",
+        ".........#+#..#+#.......",
+        "....-r-..#'#..#'#.......",
+        "....-r-..###..###.......",
+        "........................",
+        "....-r-...t....t........",
+        "....-r-.............t...",
+        "........................",
+        "........................",
+        "........................",
+        "........................"
+      ],
+      "terrain": {
+        "#": "t_grass",
+        "'": "t_grass",
+        "+": "t_grass",
+        "-": "t_dirt",
+        ".": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "b": "t_dirt",
+        "d": "t_dirt",
+        "r": "t_dirt",
+        "t": "t_tree"
+      },
+      "furniture": {
+        "#": "f_canvas_wall",
+        "'": "f_groundsheet",
+        "+": "f_canvas_door",
+        "-": "f_camp_chair",
+        "b": "f_firering",
+        "r": "f_tourist_table"
+      },
+      "place_items": [
+        { "item": "cannedfood", "x": [ 16, 4 ], "y": [ 7, 21 ], "chance": 75 },
+        { "item": "cannedfood", "x": [ 16, 4 ], "y": [ 7, 21 ], "chance": 75 },
```

Comment:
Double rolls can be achieved by `{ "item": "cannedfood", "x": [ 16, 4 ], "y": [ 7, 21 ], "chance": 75, "repeat": 2 }`